src/networking.c

```diff
@@ -860,7 +860,7 @@ void addReplyBigNum(client *c, const char* num, size_t len) {
     } else {
         addReplyProto(c,"(",1);
         addReplyProto(c,num,len);
-        addReply(c,shared.crlf);
```

Comment:
so does it mean we not longer need `shared.crlf`? and we can remove it from `createSharedObjects`

Reply:
indeed. Removed it on the latest commit.